src/routes/expenses.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const express = require('express');
+const expenseController = require('../controllers/expenses');
+const router = express.Router();
+
+router.get('/', expenseController.getAll);
```

Comment:
i think better naming are userRouter and expenseRouter for this ::))

Reply:
yep, done 👍